src/index.ts

```diff
@@ -238,13 +238,12 @@ function assertPathIsNotBanned(dirPath: string) {
 }
 
 function getTypeScriptVersionFromComment(text: string): AllTypeScriptVersion | undefined {
-    const searchString = "// TypeScript Version: ";
-    const x = text.indexOf(searchString);
-    if (x === -1) {
+    const match = text.match(/\/\/ (?:Minimum)? TypeScript Version: /);
```

Comment:
Hmm, does one of those spaces need to go inside the parens? I think this will match `// Minimum TypeScript Version: ` or `//  TypeScript Version: ` but not `// TypeScript Version` (with only one space after the comment start).

(Noticed because this seemed to cause my PR on DefinitelyTyped to start failing)

Reply:
You’re right, this was a typo. Surprised a test didn’t catch it 😬